lib/openstudio-standards/btap/NECB_building_types.csv

```diff
@@ -1,35 +1,35 @@
-﻿Activity,Density,Category,Includes,Excludes,Fallback
-arena,10,recreation,arena,,
-gym,10,recreation,gym,,
-exercise,10,commerce,exercise,,
-office,25,commerce,office,,
-retail,30,commerce,retail,,
-leisure,10,commerce,leisure/lounge/bar,fast/cafeteria/family,
-fastfood,10,commerce,fast/cafeteria,leisure/lounge/bar/family,
-restaurant,10,commerce,family/restaurant,leisure/lounge/bar/fast/cafeteria,
-hotel,25,lodging,hotel,,
-motel,25,lodging,motel,hotel,hotel
-dorm,30,housing,dorm,,
-residential,25,housing,multi/residential/apartment,health/care,
-care,25,housing,care,health/clinic/multi/residential,residential
+﻿Activity,LiveLoad,Category,Includes,Excludes,Fallback
+arena,16,recreation,arena,,
```

Comment:
Dropping _occupant densities_ (already available from spacetype data), in favour of anticipated non-occupant _liveload_.